instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -273,5 +279,9 @@ def process_response(
                 )
             )
 
+        propagator = get_global_response_propagator()
+        if propagator:
+            propagator.inject(resp, setter=self.back_propagation_setter)
```

Comment:
You are able to access a class attribute like this via `self`?

Reply:
Yes but it can lead to bugs later. Will change.